Server/Files_transfer_ownership.md

```diff
@@ -0,0 +1,35 @@
+Transfer from user1 to user2, both users are in group "group1". "User1" shares subfolder with files with "user2" and "user3" | The files/folders are transferred to user2 and they do not appear for user1 any longer | :construction:  |
+Transfer from user1 to user2, both users are in group "group1". check that share permissions are kept | The files/folders are transferred to user2 and they do not appear for user1 any longer | :construction:  |
+Transfer from user1 to user2, both users are in group "group1". "User1" shares folder with group "group1" (in which the user is member) | The files/folders are transferred to user2 and they do not appear for user1 any longer | :construction:  |
+Transfer from user1 to user2, both users are in group "group1". "User1" shares folder with group "group1" (in which the user is not member) | The files/folders are transferred to user2 and they do not appear for user1 any longer | :construction:  |
+Transfer from user1 to user2, both users are in group "group1". "User1" shares folder with group "group1", "user2" renamed target folder | The files/folders are transferred to user2 and they do not appear for user1 any longer | :construction:  |
+Transfer from user1 to user2, both users are in group "group1". "User1" has link share | The files/folders are transferred to user2 and they do not appear for user1 any longer | :construction:  |
+Transfer from user1 to user2, both users are in group "group1". "User1" is recipient of local shares => incoming shares not kept | The files/folders are transferred to user2 and they do not appear for user1 any longer | :construction:  |
+Transfer from user1 to user2, both users are in group "group1". "User1" is recipient of fed shares => incoming shares not kept | The files/folders are transferred to user2 and they do not appear for user1 any longer | :construction:  |
+Transfer from user1 to user2, both users are in group "group1". "User1" has outgoing fed shares => they are still accessible after transfer | The files/folders are transferred to user2 and they do not appear for user1 any longer | :construction:  |
+Transfer from user1 to user2, both users are in group "group1". Metadata still exists (activity, system tags, comments) | The files/folders are transferred to user2 and they do not appear for user1 any longer | :construction:  |
+Transfer from user1 to user2, both users are in group "group1". Favorite information not kept for target user (?) | The files/folders are transferred to user2 and they do not appear for user1 any longer | :construction:  |
+Transfer from user1 to user2, both users are in group "group1". Non-decrypted encrypted files | The files/folders are transferred to user2 and they do not appear for user1 any longer | :construction:  |
```

Comment:
Before adding encryption tests I'd add a row describing the environment. "Server with encryption enabled" or something similar.